types/handsontable/index.d.ts

```diff
@@ -80,6 +80,7 @@ declare namespace __Handsontable {
         setCellMeta(row: number, col: number, key: string, val: string): void;
         setCellMetaObject(row: number, col: number, prop: Object): void;
         setDataAtCell(row: number|any[], col: number, value: string, source?: string): void;
+        setDataAtCell(changes: [number, number, any][]): void;
```

Comment:
This is an intentional overload method. Both signatures are valid according to the documentation: https://docs.handsontable.com/pro/1.10.2/Core.html#setDataAtCell